src/components/ExtraPages/Help/Contents/DataInfo.js

```diff
@@ -0,0 +1,56 @@
+// Information to populate the Data Help page
+
+export const DataInfoHead = {
+  title: "Information on Data Sources",
+  intro: [
+    "The main sources of climate information in this plot explorer are the EuroCORDEX and UKCP18 regional ensembles, but several other sources are also used: brief descriptions of these are provided here. All indices are first computed on each model's native grid, then ",
+    "regridded to a common 12km OSGB grid to facilitate comparisons.",
+  ],
+};
+
+export const DataInfoContents = [
+  {
+    source: "CMIP5-13 ensemble",
+    description:
+      "This consists of simulations from thirteen of the coupled ocean-atmosphere models run as part of the Coupled Model Intercomparison Project Phase 5 (CMIP5) experiment. This ensemble was constructed to supplement the 60km projections in the UKCP18 global projections, with the constituent models selected on the basis of performance and diversity criteria to in order to increase the range of global and regional changes sampled. The members of the CMIP5-13 ensemble are listed in the Table 1.",
```

Comment:
```suggestion
      "This consists of simulations from thirteen of the coupled ocean-atmosphere models run as part of the Coupled Model Intercomparison Project Phase 5 (CMIP5) experiment. This ensemble was constructed to supplement the 60km projections in the UKCP18 global projections, with the constituent models selected on the basis of performance and diversity criteria to in order to increase the range of global and regional changes sampled. The members of the CMIP5-13 ensemble are listed in Table 1.",
```

CMIP5 should also link to https://www.wcrp-climate.org/wgcm-cmip/wgcm-cmip5